.travis.yml

```diff
@@ -59,7 +59,8 @@ script:
     - pytest -s
 
     # Clone kotekan and run some of its tests
-    - ./kotetest.sh
+    - mkdir skyfield_data
+    - CAPUT_SKYFIELD_PATH=skyfield_data ./kotetest.sh
```

Comment:
noooooo

Reply:
I actually don't think this was the problem. One of the two CI builds still failed. It doesn't say why and I can't reproduce...

Reply:
test_dataset_broker.py:59: AssertionError

============================== 1 failed in 3.26s ===============================

The command "CAPUT_SKYFIELD_PATH=skyfield_data ./kotetest.sh" exited with 1.

Reply:
Oh, yeah, this is a super vague assert

```
>               assert call([producer_path, "--", port]) == 0

E               AssertionError: assert -4 == 0

E                +  where -4 = call(['../build/tests/boost/dataset_broker_producer', '--', '46083'])
```

Reply:
Need to print stdout AND stderr also in case it fails...